src/composite.c

```diff
@@ -0,0 +1,119 @@
+#include <stdio.h>
+#include <vips/vips.h>
+
+/* Composite images `src` and `dst`.
+ */
+int Composite(VipsObject *context, VipsImage *src, VipsImage *dst,
```

Comment:
:+1: to making it more familiar for JavaScript developers. Is `PascalCase` more common for C++ than `camelCase`?